x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -715,10 +715,11 @@ export class TaskRunner<
 
       const { errors: errorsFromLastRun } = this.ruleResult.getLastRunResults();
       if (errorsFromLastRun.length > 0) {
+        const isUserError = !errorsFromLastRun.some((lastRunError) => !lastRunError.userError);
         return {
           taskRunError: createTaskRunError(
             new Error(errorsFromLastRun.join(',')),
```

Comment:
**question:** do we need to map here as well?

```suggestion
            new Error(errorsFromLastRun.map((lastRunError) => lastRunError.message).join(',')),
```

Reply:
Good catch, fixed.